common/persistence/cassandra/cassandraHistoryPersistence.go

```diff
@@ -122,6 +123,9 @@ func (h *cassandraHistoryV2Persistence) AppendHistoryNodes(
 		batch.Query(v2templateUpsertData,
 			branchInfo.TreeId, branchInfo.BranchId, request.NodeID, request.TransactionID, request.Events.Data, request.Events.EncodingType.String())
 		err = h.session.ExecuteBatch(batch)
+		h.logger.Error("####### BATCH QUERY ERROR")
+		h.logger.Error(prettyPrint(batch.Entries), tag.Error(err))
```

Comment:
Can you also log workflow and run ID?

Reply:
workflow ID & run ID is not accessible from here